docs/conf.py

```diff
@@ -277,3 +290,9 @@ def setup(app):
 
 # show inherited members
 # autodoc_default_flags = ['members', 'undoc-members', 'inherited-members', 'show-inheritance']
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+
+if read_the_docs_build:
+
+    subprocess.call('cd ../minorminer/docs/; make cpp', shell=True)
```

Comment:
Is there any reason not to put this in [`.readthedocs.yml`](https://docs.readthedocs.io/en/stable/config-file/v2.html)?

I would not recommend we run `make` on module import. :see_no_evil:

Reply:
Ups. On a closer look, RtD doesn't support custom (pre-)build steps. They do have a [feature request opened](https://github.com/readthedocs/readthedocs.org/issues/6662), likely to be implemented at some point.

I expected it's possible to customize the [requirements install](https://docs.readthedocs.io/en/stable/config-file/v2.html#python) a bit more. Also, to use a custom docker image. But no. Pretty disappointing. :stuck_out_tongue: 

There is [this hackish workaround](https://github.com/readthedocs/readthedocs.org/issues/6662#issuecomment-653675848) to put a build step in a separate yaml file, but I'm not sure that's any better than running from `conf.py`.

Reply:
Thanks @randomir!